operator/v1/types_ingress.go

```diff
@@ -362,6 +362,8 @@ var (
 	// DNSReady indicates the ready state of any DNS records for the ingress
 	// controller.
 	DNSReadyIngressConditionType = "DNSReady"
+	// NodeReady indicates the node state for the ingress controller.
+	NodeReadyIngressConditionType = "NodeReady"
```

Comment:
We've been going towards the pattern `FooDegraded`; how about `NodeDegraded`?

Reply:
OK. I'll update after we settle on the approach for solving this problem.